### PR TITLE
[MM-15027] Consistent urls in System Console, fix urls in schemas

### PR DESCRIPTION
**Pull request by bradjcoughlin**
#### Summary
Cleans up System Console URL structure (eg `/admin-console/<section>/<page>`), fixes references to pages and links in admin definition schema such as help text.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-15027

#### Related Pull Requests
N/A

### CODE REVIEW

File: components/admin_console/admin_definition.jsx [hunk 1]
```diff
@@ -512,11 +515,11 @@ export default {
                                 </b>
                             ),
                             recycleDatabaseConnections: (
-                                <a href='../advanced/database'>
+                                <a href='../environment/database'>
                                     <b>
                                         <FormattedMessage
                                             id='admin.reload.reloadDescription.recycleDatabaseConnections'
-                                            defaultMessage='Database > Recycle Database Connections'
+                                            defaultMessage='Environment > Database > Recycle Database Connections'
```
Comment (saturninoabril) [inline components/admin_console/admin_definition.jsx]: Should update corresponding translation text at i18n/en.json.

Same case for `admin.support.enableTermsOfServiceHelp`, `admin.ldap.groupFilterFilterDesc` and other help texts that were changed.

Comment (bradjcoughlin) [reply components/admin_console/admin_definition.jsx]: I'll re-check those, thanks @saturninoabril

File: components/admin_console/__snapshots__/admin_console.test.jsx.snap [hunk 2]
```diff
@@ -18,138 +18,138 @@ exports[`components/AdminConsole should generate the routes 1`] = `
     >
       <Switch>
         <Route
-          key="license"
-          path="/license"
+          key="about/license"
+          path="/about/license"
```
Comment (sudheerDev) [inline components/admin_console/__snapshots__/admin_console.test.jsx.snap]: We will be breaking existing urls if we don't add redirection for old URl's to the new ones.

Ex: If we have old urls in posts they wont be working anymore. Are we ok with this?.

Comment (jwilander) [reply components/admin_console/__snapshots__/admin_console.test.jsx.snap]: We're generally OK with that. I don't think there's any places in the documentation where we link to specific system console pages. The fact is the whole page is getting re-organized so some of the old links won't work and that's ok as long as we update any links inside the product to point to the appropriate pages.

cc @wiersgallak

Comment (sudheerDev) [reply components/admin_console/__snapshots__/admin_console.test.jsx.snap]: I meant more in the lines of like plugins which use these URL's(I remember hearing NPS plugin using the url) or in existing posts in channel with the old URL's.

Comment (sudheerDev) [reply components/admin_console/__snapshots__/admin_console.test.jsx.snap]: Never mind. The chances of anyone depending on these URL's are really slim.

**Comment by sudheerDev** [comment]
Sorry i did a quick search and noticed few places where we need to make the changes in our repo to not break these url's.

https://github.com/mattermost/mattermost-webapp/blob/4cfbc007bf59343bd0dcc4170e9a4a4480eee3bd/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/permission_team_scheme_settings.jsx#L322

https://github.com/mattermost/mattermost-webapp/blob/4cfbc007bf59343bd0dcc4170e9a4a4480eee3bd/components/admin_console/permission_schemes_settings/permission_team_scheme_settings/permission_team_scheme_settings.jsx#L405

https://github.com/mattermost/mattermost-webapp/blob/4cfbc007bf59343bd0dcc4170e9a4a4480eee3bd/components/audit_table/audit_table.jsx#L562

**Comment by bradjcoughlin** [comment]
@sudheerDev fixed the various routes connected to the permissions pages.

**Comment by sudheerDev** [comment]
@bradjcoughlin Assigning you for merging it as i don't know if this for 5.11v or 5.12?

**Pull request merged**